src/main/java/oneus/GSMATCH/global/level/Point.java

```diff
@@ -0,0 +1,23 @@
+package oneus.GSMATCH.global.level;
+
+import lombok.RequiredArgsConstructor;
+import oneus.GSMATCH.domain.user.entity.UserEntity;
+import oneus.GSMATCH.domain.user.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class Point {
+    private final UserRepository userRepository;
+    private final Level level;
+
```

Comment:
```suggestion
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

@Service
@RequiredArgsConstructor
public class Point {
    private final UserRepository userRepository;
    private final Level level;

    @Transactional
```

해당 작업은 데이터 변경이 있는 작업이여서 @Transactional 어노테이션이 필요할 것 같습니다